content/source/docs/enterprise/vcs/index.html.md

```diff
@@ -14,6 +14,19 @@ Terraform Enterprise (TFE) is more powerful when you integrate it with your vers
 
 We recommend configuring VCS access when first setting up a TFE organization, and you might need to add additional VCS providers later depending on how your organization grows.
 
+## Supported VCS Providers
+
+TFE supports the following VCS providers:
+
+- [GitHub](./github.html)
+- [GitHub Enterprise](./github-enterprise.html)
+- [GitLab.com](./gitlab-com.html)
+- [GitLab EE and CE](./gitlab-eece.html)
+- [Bitbucket Cloud](./bitbucket-cloud.html)
+- [Bitbucket Server](./bitbucket-server.html)
+
+Use the links above to see details on configuring VCS access for each supported provider. If you use another VCS that is not supported, you can build an integration via [the API-driven run workflow](../run/api.html).
+
```

Comment:
I'm wondering if we should replace "is not supported" with "is not in the list above" ?